exercises/concept-wip/arys-amazing-lasagna/src/arys_amazing_lasagna.cr

```diff
@@ -0,0 +1,15 @@
+class Lasagna
+  # TODO: define the 'EXPECTED_MINUTES_IN_OVEN' constant
+
+  def self.remaining_minutes_in_oven(actual_minutes_in_oven)
+    raise "Please implement the Lasagna#remaining_minutes_in_oven method"
+  end
+
+  def self.preparation_time_in_minutes(layers)
+    raise "Please implement the Lasagna#preparation_time_in_minutes method"
+  end
+
+  def self.total_time_in_minutes(number_of_layers, actual_minutes_in_oven)
+    raise "Please implement the Lasagna#total_time_in_minutes method"
+  end
```

Comment:
In the bools PR, you said you don't want to introduce `self` just yet. The tests also don't expect `self` to be used.
```suggestion
  def remaining_minutes_in_oven(actual_minutes_in_oven)
    raise "Please implement the Lasagna#remaining_minutes_in_oven method"
  end

  def preparation_time_in_minutes(layers)
    raise "Please implement the Lasagna#preparation_time_in_minutes method"
  end

  def total_time_in_minutes(number_of_layers, actual_minutes_in_oven)
    raise "Please implement the Lasagna#total_time_in_minutes method"
  end
```